cpp/ps0001/p1Solution.cpp

```diff
@@ -1,19 +1,17 @@
-#include<iostream>
-using namespace std;
-double CalculateSum(double num1, double num2, double num3) {
-	return num1 + num2 + num3;
-}
-int main() {
+#include <iostream>
 
-	cout << "The sum of your 3 numbers is: " << CalculateSum(155, 940, 168);
+using namespace std;
 
-	/*
-	double num1, num2, num3, sum;
-	cout<< " Enter 3 number to sum:" ;
+int main() {
+	// promptiung user for 3 numbers to add 1/2
+	short num1, num2, num3, sum;
+	cout << "Enter 3 number to sum:" ;
 	cin >> num1 >> num2 >> num3;
+
+	// outputting the result of the summation process 3/3
 	sum = num1 + num2 + num3;
-	cout << "The of number is:" << sum;*/
+	cout << "The of number is:" << sum;

```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [ ] output is as required
```
155
940
168
--------
1263
```
+1 bonus mark as your program is user friendly